docs/src/content/docs/de/resources/community-content.mdx

```diff
@@ -119,6 +119,21 @@ Erkunde die von der Community erstellten Inhalte, die von Starlight-Benutzern ge
 		title="Lesefortschrittsanzeige hinzufügen"
 		description="Erfahre, wie du eine horizontale Lesefortschrittsanzeige zu deiner Starlight-Website hinzufügen kannst."
 	/>
+	<LinkCard
+		href="https://blog.trueberryless.org/blog/starlight-sidebar-whitespace/"
+		title="Verpass deiner Starlight-Seitenleiste eigene Stile"
```

Comment:
Is this translation correct? I’ve never seen this use of “verpassen” before.

Maybe? “Verwende benutzerdefinierte Stile in deiner Starlight-Seitenleiste”? Or “Deine Starlight-Seitenleiste anpassen mit benutzerdefinierte Stile”?